h2o-core/src/main/java/water/udf/ColumnBase.java

```diff
@@ -0,0 +1,26 @@
+package water.udf;
+
+import water.Iced;
+import water.fvec.Frame;
+import water.fvec.Vec;
+
+/**
+ * Basic common behavior for Functional Columns
+ */
+public abstract class ColumnBase<T> extends Iced<ColumnBase<T>> implements Column<T> {
+
+  public abstract T get(long idx);
+  
+  @Override public T apply(long idx) { return get(idx); }
+
+  @Override public T apply(Long idx) { return get(idx); }
+
+  @Override 
+  public boolean isCompatibleWith(Column<?> ys) {
+    boolean itis = new Frame(vec()).isCompatible(new Frame(ys.vec()));
```

Comment:
Can't we just make the `isCompatibleWith` from `Vec` public and use that? Creating a whole `Frame` seems unnecessarily heavy. Also why aren't we just returning `itis` here?